src/Network/Network/help/Get-AzNetworkManagerEffectiveConnectivityConfigurationList.md

```diff
@@ -0,0 +1,151 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
+schema: 2.0.0
+---
+
+# Get-AzNetworkManagerEffectiveConnectivityConfigurationList
+
+## SYNOPSIS
+Lists NetworkManager Effective Connectivity Configurations applied on a virtual networks.
+
+## SYNTAX
+
+```
+Get-AzNetworkManagerEffectiveConnectivityConfigurationList -VirtualNetworkName <String>
+ -VirtualNetworkResourceGroupName <String> [-SkipToken <String>] [-DefaultProfile <IAzureContextContainer>]
```

Comment:
We received customer feedback that this was confusing when named 'ResourceGroup', because the cmdlet is related to network manager, but we are not asking for the network manager resource group. Instead, we want the virtual network resource group, so we can return info about the network manager configs applied on the vnet.